README.md

```diff
@@ -340,4 +343,4 @@ Running `lerna` without arguments will show all commands/options.
 
 - `lerna`: the current version of `lerna` being used.
 - `version`: the current version of the repository.
-- `publishConfig.ignore`: an array of globs that won't be included in `lerna updated/publish`. Use this to prevent publishing a new version just because of `README.md` typo.
+- `publishConfig.ignore`: an array of globs that won't be included in `lerna updated/publish`. Use this to prevent publishing a new version unnecessarily for changes, such as fixing a `README.md` typo.
```

Comment:
Tried to have this be a more general description while still including the specific example